### ISSUE TITLE
template function not working

### ISSUE THREAD

**Issue by isaacrayner** [PJTewkesbury/MMM-WebHookAlerts#3]
Hi there. When using the template field, the content between the quotes gets inserted as the message. See image below. No variation on the information provided in the documentation is able to change this. Is it no longer working?
I have tested using various template designs, but only the {{message}} content works anymore.

![image](https://user-images.githubusercontent.com/86776303/172083719-52e403a4-b864-4ea7-bb8e-2f93bd0c71dd.png)

**Comment by isaacrayner** [comment]
Here is example code for the template:					
```
templateName: "SimpleAlert2",
				template: "<div class='fullscreen' style='border:1px solid black;'><b>{{message}}</b></div>",
				title  : "{{Title}}",
				displaySeconds:30,
				fadeSpeed:10,
```

**Comment by isaacrayner** [comment]
womp womp womp. no one here anymore.

**Comment by sigel** [comment]
@isaacery02 Yeah too bad... same problem here... also breaks MMM-Scenes for some reason.

**Comment by jtolksdo** [comment]
same here :-(
